common/src/main/java/com/wynntils/models/wynnitem/parsing/WynnItemParser.java

```diff
@@ -55,6 +57,9 @@ public final class WynnItemParser {
 
     public static final Pattern SET_BONUS_PATTEN = Pattern.compile("^§aSet Bonus:$");
 
+    // Test suite: https://regexr.com/7i5h5
+    public static final Pattern SHINY_STAT_PATTERN = Pattern.compile("^§f⬡ §7([a-zA-Z ]*?): §f(\\d+)$");
```

Comment:
What does the `?` do after the `*`? Also, I'd suggest using `+` instead of `*`; we'd want at least one character here.

Reply:
okay the unneeded char in the regex has been removed 👍